### PR TITLE
First Draft.

### PR DESCRIPTION
**Pull request by Piwington**

### CODE REVIEW

File: DataGen.py [hunk 1]
```diff
@@ -157,7 +157,44 @@ def main(argv):
...
+                  '-o <output_file> -i <starting_id> -p <period_split>')
+            sys.exit()
+        elif opt in ("-i", "--input_file"):
+            input_file_name = arg
+        elif opt in ("-r", "--region_file"):
+            region_file_name = arg
+        elif opt in ("-c", "--county_file"):
+            county_file_name = arg
+
+    input_df = pd.read_csv(input_file_name + '.csv')
```
Comment (kingmushroom) [inline DataGen.py]: hard to tell on git. Do these jave defaults?

File: LookupGen.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import json
+import random
+import sys
+
+import pandas as pd
+
+
+def main(argv):
+
+    input_file_name = 'fixtures/outputs/sample_survey_all'
```
Comment (kingmushroom) [inline LookupGen.py]: might be missing file extension on this line

Comment (kingmushroom) [reply LookupGen.py]: Nope, i see now

File: LookupGen.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+    with open(county_file_name, "r") as county_file:
+        county_json = json.load(county_file)
+    county_df = pd.DataFrame(county_json)
+
+    id_column = input_df[['responder_id', 'gor_code', 'land_or_marine']].drop_duplicates()
+    prepared_df = pd.merge(id_column, region_df[['region', 'gor_code']],
+                           on='gor_code', how="left")
+
+    prepared_df.to_json("test_lookup.json", orient="records")
```
Comment (KrisRogos) [inline LookupGen.py]: Why dumping to test_lookup.json before the work is complete?

File: LookupGen.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    print(' Complete')
+    print('----------')
+
+
+def get_county(row, county_df):
+    toggle = ''
+    if row['land_or_marine'] == 'L':
+        toggle = 'n'
+    elif row['land_or_marine'] == 'M':
+        toggle = 'y'
```
Comment (KrisRogos) [inline LookupGen.py]: Will this work for not sand & gravel surveys

File: DataGen.py (hunk 1 above)
Comment (Piwington) [reply DataGen.py]: Now they do.

File: LookupGen.py (hunk 3 above)
Comment (Piwington) [reply LookupGen.py]: Removed Test Line.

File: LookupGen.py (hunk 4 above)
Comment (Piwington) [reply LookupGen.py]: Sand And Gravel Are The Only Survey We Make This Lookup For.

File: fixtures/region_lookup.csv [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+CB,3,North West
+DC,2,Pigside
+ED,6,East Westlands
+FE,5,West Eastlands
+GF,7,Eastern
+GG,7,Eastern
+HH,9,South East
+JG,9,South East
+KJ,10,South West
+WW,11,Hills Of Death
```
Comment (kingmushroom) [inline fixtures/region_lookup.csv]: Hills of dampness?

**Pull request merged**